src/test/java/com/google/devtools/build/lib/packages/util/mock/osx_cc_toolchain_config.bzl

```diff
@@ -2319,6 +2330,7 @@ def _impl(ctx):
         )
     elif (ctx.attr.cpu == "darwin_x86_64"):
         objc_compile_action = action_config(
+            enabled = True,
```

Comment:
Why are these needed?

Reply:
this is what objc_library now uses to tell if it's supported to build objc